{{cookiecutter.python_name}}/.github/workflows/build.yml

```diff
@@ -9,40 +9,62 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.9']
+
     steps:
     - name: Checkout
       uses: actions/checkout@v2
 
     - name: Base Setup
-        uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
+      uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
 
     - name: Install dependencies
       run: python -m pip install -U jupyterlab~=3.1 check-manifest
 
-    - name: Build the extension
+    - name: Lint the extension
       run: |
         set -eux
         jlpm
         jlpm run eslint:check
-        python -m pip install .
+    
+    - name: Test the extension
+      run: |
+        set -eux
+        jlpm run test
+
+    - name: Build the extension
+      run: |
+        set -eux
+        python -m pip install .[test]
 {% if cookiecutter.kind.lower() == 'server' %}
+        pytest -vv -r ap --cov {{ cookiecutter.python_name }}
+        jupyter server extension list
         jupyter server extension list 2>&1 | grep -ie "{{ cookiecutter.python_name }}.*OK"
 {% endif %}
+        jupyter labextension list
         jupyter labextension list 2>&1 | grep -ie "{{ cookiecutter.labextension_name }}.*OK"
         python -m jupyterlab.browser_check
 
+    - name: Package the extension
+      run: |
+        set -eux
         check-manifest -v
 
         pip install build
-        python -m build --sdist
+        python -m build
+        cp dist/*.tar.gz myextension.tar.gz
```

Comment:
users will _generally_ be installing the `.whl` artifact: testing the sdist is good, but it's really both

Reply:
:+1: